app/Community/Components/UserProfileMeta.php

```diff
@@ -28,58 +29,255 @@ public function __construct(
+        $codeNotesCreatedStat = [
+            'label' => 'Code notes created',
+            'value' => localized_number($totalAuthoredCodeNotes),
+            'href' => $totalAuthoredCodeNotes ? '/individualdevstats.php?u=' . $user->User . '#code-notes' : null,
+            'isMuted' => !$totalAuthoredCodeNotes,
+        ];
+
+        // Leaderboards created
+        $totalAuthoredLeaderboards = $user->authoredLeaderboards()
+            ->select(DB::raw('COUNT(LeaderboardDef.ID) AS TotalAuthoredLeaderboards'))
+            ->value('TotalAuthoredLeaderboards');
+        $leaderboardsCreatedStat = [
+            'label' => 'Leaderboards created',
+            'value' => localized_number($totalAuthoredLeaderboards),
+            'href' => $totalAuthoredLeaderboards ? '/individualdevstats.php?u=' . $user->User : null,
+            'isMuted' => !$totalAuthoredLeaderboards,
+        ];
+
+        // Open tickets
+        $openTickets = null;
+        if ($user->ContribCount) {
+            $openTickets = array_sum(countOpenTicketsByDev($user->User));
+        }
+        $openTicketsStat = [
+            'label' => 'Open tickets',
+            'value' => $openTickets === null ? "Tickets can't be assigned to {$user->User}." : localized_number($openTickets),
+            'href' => $openTickets ? '/ticketmanager.php?u=' . $user->User : null,
+            'isMuted' => !$openTickets,
+        ];
+
+        return compact(
+            'achievementsUnlockedByPlayersStat',
+            'codeNotesCreatedStat',
+            'leaderboardsCreatedStat',
+            'openTicketsStat',
+            'pointsAwardedToPlayersStat',
+            'setsWorkedOnStat',
+        );
+    }
+
+    private function buildHardcorePlayerStats(User $user, array $userMassData, array $hardcoreRankMeta): array
+    {
+        $hardcorePoints = $userMassData['TotalPoints'] ?? 0;
+        $weightedPoints = $userMassData['TotalTruePoints'] ?? 0;
+        $retroRatio = $weightedPoints ? sprintf("%01.2f", $weightedPoints / $hardcorePoints) : null;
+
+        // Hardcore points
+        $hardcorePointsStat = [
+            'label' => 'Points',
+            'value' => localized_number($hardcorePoints),
+            'weightedPoints' => $userMassData['TotalTruePoints'],
+            'isMuted' => !$hardcorePoints,
+        ];
+
+        // Hardcore site rank
+        $hardcoreSiteRankValue = '';
+        if ($userMassData['Untracked']) {
+            $hardcoreSiteRankValue = 'Untracked';
+        } elseif ($hardcorePoints > 0 && !$hardcoreRankMeta['rank']) {
+            $hardcoreSiteRankValue = 'requires ' . Rank::MIN_POINTS . ' points';
+        } elseif ($hardcorePoints === 0 && !$hardcoreRankMeta['rank']) {
+            $hardcoreSiteRankValue = 'none';
+        } else {
+            $hardcoreSiteRankValue = "#" . localized_number($hardcoreRankMeta['rank']) . " of " . localized_number($hardcoreRankMeta['numRankedUsers']);
```

Comment:
I imagine it's hard to do with the new construction paradigm, but I prefer just having the rank linked.
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/9a58f0ea-3654-4d61-a9d3-fec3a01d7d5f)
vs
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/c73b3a12-22d1-40fe-a732-fcce2a2cb53b)

It seems like you've got something working - White Points appear differently hand normal hardcore points.

Reply:
Addressed in latest.